storage/storage_src.go

```diff
@@ -42,8 +42,14 @@ type storageImageSource struct {
 	getBlobMutex    sync.Mutex              // Mutex to sync state for parallel GetBlob executions
```

Comment:
We should precisely document which fields are handled by `getBlobMutex`.

Typically in c/image, we’d move it into a separate “section”, starting with `getBlobMutex`, and then say “all following fields can only be accessed with lock held”. Or alternatively,
```go
getBlobMutex sync.Mutex
getBlobMutexProtected struct {
// … protected fields
}
```
but that makes initialization of the containing structure annoying, we’d need to name the embedded type.

Reply:
thanks, fixed